### DISCUSSION TITLE
Label Color & Size

### DISCUSSION THREAD

**Discussion by frxldi-xyz** [kurkle/chartjs-chart-sankey#98] [Q&A]
I love this library. I have some question, there is any option to change Label Size and Color? 
I already see the example but not found what related to that

Thank you

**Comment by stockiNail** [comment]
@frxldi-xyz you can add `font` and `color` options to the dataset to change color and font to the labels.
AFAIK they are NOT currently "scriptable".

Codepen: https://codepen.io/stockinail/pen/poxXwLJ

**Reply by frxldi-xyz** [reply]
Thankyou for the answer. Finally found it
